src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx

```diff
@@ -172,37 +172,39 @@ export const View = (props: Props) => {
           }
         );
       }
-      return (
-        <>
-          {isScanResult(exposure) ? (
-            // Scanned result
-            <li key={index} className={styles.exposureListItem}>
-              <ExposureCard
-                exposureData={exposure}
-                exposureName={exposure.data_broker}
-                exposureDetailsLink={exposure.link}
-                dateFound={dateObject(exposure.created_at)}
-                statusPillType="needAction"
-                locale={props.locale}
-                color={getRandomLightNebulaColor(exposure.data_broker)}
-              />
-            </li>
-          ) : (
-            // Breaches result
-            <li key={index} className={styles.exposureListItem}>
-              <ExposureCard
-                exposureData={exposure}
-                exposureName={exposure.Name}
-                fromEmail={email}
-                exposureDetailsLink={""}
-                dateFound={exposure.AddedDate}
-                statusPillType="needAction"
-                locale={props.locale}
-                color={getRandomLightNebulaColor(exposure.Name)}
-              />
-            </li>
-          )}
-        </>
+      return isScanResult(exposure) ? (
+        // Scanned result
+        <li
+          key={`scan-${exposure.id}-${index}`}
+          className={styles.exposureListItem}
+        >
+          <ExposureCard
+            exposureData={exposure}
+            exposureName={exposure.data_broker}
+            exposureDetailsLink={exposure.link}
+            dateFound={dateObject(exposure.created_at)}
+            statusPillType="needAction"
+            locale={props.locale}
+            color={getRandomLightNebulaColor(exposure.data_broker)}
+          />
+        </li>
+      ) : (
+        // Breaches result
+        <li
+          key={`breach-${exposure.Id}-${index}`}
+          className={styles.exposureListItem}
+        >
+          <ExposureCard
+            exposureData={exposure}
+            exposureName={exposure.Name}
```

Comment:
What's this? The capitalization is "Weird" which makes me think it comes from HIBP; but wanted to confirm we aren't displaying the `Name` (since we typically only display the `breach.Title` to end users).

Reply:
Correct, that’s data coming from HIBP. Updated to use `.Title` instead of `.Name` in [56f344b](https://github.com/mozilla/blurts-server/pull/3246/commits/56f344b20716a565a39a0e5a5a6214415ccf9a77).